src/lib/integration_test_cloud_engine/coda_automation.ml

```diff
@@ -109,7 +109,10 @@ module Network_config = struct
     (* GENERATE ACCOUNTS AND KEYPAIRS *)
     let num_block_producers = List.length block_producers in
     let block_producer_keypairs, runtime_accounts =
-      let keypairs = Array.to_list (Lazy.force Sample_keypairs.keypairs) in
+      (* the first keypair is the genesis winner and is assumed to be untimed. Therefore dropping it, and not assigning it to any block producer *)
+      let keypairs =
+        List.drop (Array.to_list (Lazy.force Sample_keypairs.keypairs)) 1
+      in
       if List.length block_producers > List.length keypairs then
```

Comment:
can use `num_block_producers` here, rather than recalculating length

Reply:
what do you mean?

Reply:
oh i see what you mean, sure